### PR TITLE
Update 4.0.0

### PR DESCRIPTION
**Pull request by pavog**
I updated the generated code with openapi generator 7.8.0.

Changes:
- Changes return type for `APIKeysApi->getKeys()` from `\Aternos\HangarApi\Model\ApiKey` to `\Aternos\HangarApi\Model\ApiKey[]`
- Removes parameter `platform_version` from `VersionsAPI->listVersions()`
- Removes parameter `platformVersion` from `CompactProject->getVersions()`
- Removes parameter `platformVersion` from `Project->getVersions()`
- Removes member `platformVersion` from `VersionSearchOptions`
- Adds property `description` to `ProjectCompact`
- Adds model for Tag
- Changes type of property `tags` in `ProjectSettings` from `string[]` to `\Aternos\HangarApi\Model\Tag[]`

### CODE REVIEW

File: lib/Client/HangarAPIClient.php [hunk 1]
```diff
@@ -338,8 +338,7 @@ public function getProjectVersions(
             $options->getPagination(),
             $options->isIncludeHiddenChannels(),
             $options->getChannel(),
-            $options->getPlatform()?->value,
-            $options->getPlatformVersion(),
+            $options->getPlatform()?->value
```
Comment (JulianVennen) [inline lib/Client/HangarAPIClient.php]: Why did you remove the trailing comma here?

File: lib/Model/ProjectSettings.php [hunk 2]
```diff
@@ -59,7 +59,7 @@ class ProjectSettings implements ModelInterface, ArrayAccess, \JsonSerializable
       */
     protected static $openAPITypes = [
         'links' => '\Aternos\HangarApi\Model\LinkSection[]',
-        'tags' => 'string[]',
+        'tags' => '\Aternos\HangarApi\Model\Tag[]',
```
Comment (JulianVennen) [inline lib/Model/ProjectSettings.php:62]: string[] is the correct type here but editing generated files is annoying, so just leave it, I guess

**Review by JulianVennen — APPROVED** [review]

**Comment by JulianVennen** [comment]
You could also add a real enum for Tag and use it in ProjectSearchOptions, like with the Platform and ProjectCategory enums in lib/Client/Options

**Comment by pavog** [comment]
> You could also add a real enum for Tag and use it in ProjectSearchOptions, like with the Platform and ProjectCategory enums in lib/Client/Options

Done

**Review by JulianVennen — APPROVED** [review]
